src/css/application.css

```diff
@@ -1 +1,279 @@
-/* application styles */
\ No newline at end of file
+.application-section .section-description {
```

Comment:
in my opinion, we should add this to the main class or void these additional changes. Cause all this descriptions have the same logic and styles

Reply:
In design every single description has different width

Reply:
I think it's about the length of the words. we can just use max-width 709px for all descriptions on the desktop. 
What do you think?